tests/jballerina-integration-test/src/test/resources/identifier/identifier_literal_service.bal

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+listener http:Listener testEP = new(9090);
+
+@http:ServiceConfig {
```

Comment:
check without giving paths

Reply:
User is expected to provide path or encoded name when having reserved characters in the path